src/main/java/com/premiumminds/webapp/wicket/bootstrap/datepicker/BootstrapDatePickerBehaviour.java

```diff
@@ -36,9 +36,9 @@
 public class BootstrapDatePickerBehaviour extends Behavior {
 	private static final long serialVersionUID = 6150624915791893034L;
 
-	private static final ResourceReference DATE_PICKER_CSS = new CssResourceReference(BootstrapDatePickerBehaviour.class, "datepicker.css");
+	private static final ResourceReference DATE_PICKER_CSS = new CssResourceReference(BootstrapDatePickerBehaviour.class, "datepicker.min.css");
```

Comment:
I believe wicket does this for us:

By default when our application runs in DEPLOYMENT mode ContextRelativeResourceReference will automatically load the minified version of the specified resource using 'min' as postfix. In the example above it will load '/misc/js/jscript.min.js'.

https://ci.apache.org/projects/wicket/guide/8.x/single.html#_context_relative_resources